code/modules/plumbing/plumbers/synthesizer.dm

```diff
@@ -61,8 +63,25 @@
 		return
 	if(reagents.total_volume >= amount*delta_time*0.5) //otherwise we get leftovers, and we need this to be precise
 		return
+	if(volume_left < amount || !volume_left)
+		return
 	reagents.add_reagent(reagent_id, amount*delta_time*0.5)
+	volume_left -= amount
+
+/obj/machinery/plumbing/synthesizer/attackby(obj/item/O, mob/user, params)
+	if(volume_left == 200)
```

Comment:
Hard-coded volume 💀

Reply:
Also it gets displayed by every single attack.

Reply:
Oops, fixing